libminifi/test/unit/ProcessSessionTests.cpp

```diff
@@ -28,22 +28,6 @@
 
 namespace {
 
-class DummyProcessor : public minifi::core::Processor {
-  using minifi::core::Processor::Processor;
-
- public:
-  static constexpr const char* Description = "A processor that does nothing.";
-  static auto properties() { return std::array<core::Property, 0>{}; }
-  static auto relationships() { return std::array<core::Relationship, 0>{}; }
-  static constexpr bool SupportsDynamicProperties = false;
-  static constexpr bool SupportsDynamicRelationships = false;
-  static constexpr core::annotation::Input InputRequirement = core::annotation::Input::INPUT_ALLOWED;
-  static constexpr bool IsSingleThreaded = false;
-  ADD_COMMON_VIRTUAL_FUNCTIONS_FOR_PROCESSORS
-};
-
-REGISTER_RESOURCE(DummyProcessor, Processor);
```

Comment:
This is a good idea.  There are two more identical copies of this dummy processor, in `ContentRepositoryDependentTests.h` and `GCPCredentialsControllerServiceTests.cpp`; can you get rid of them, too, please?

Reply:
Updated in 6da42b94abc78aaa874eaba0258bf07c49d21660